src/reducers/localeFactory.js

```diff
@@ -2,17 +2,20 @@ import Immutable from "immutable";
 import { CHANGE_LOCALE, GET_CULTURES_SUCCESS } from "../actions/locale";
 
 const localeFactory = supportedLocales => {
```

Comment:
Documentation actionsreducersselectors.md is not updated (Now it 'An array of IETF language tags')